### PR TITLE
Add Python code sample for Cloud Functions + Memorystore for Redis

### PR DESCRIPTION
**Pull request by ericschmidtatwork**
I adapted the existing sample from https://cloud.google.com/memorystore/docs/redis/connect-redis-instance-flex.

### CODE REVIEW

File: functions/memorystore/redis/main.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# [START functions_memorystore_redis]
+
+import os
+import redis
+
+redis_host = os.environ.get('REDISHOST', 'localhost')
+redis_port = int(os.environ.get('REDISPORT', 6379))
+redis_client = redis.StrictRedis(host=redis_host, port=redis_port)
+
+def visit_count(request):
```
Comment (gguuss) [inline functions/memorystore/redis/main.py]: Add a newline here for lint

File: functions/memorystore/redis/main.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  value = redis_client.incr('visits', 1)
```
Comment (gguuss) [inline functions/memorystore/redis/main.py]: These lines (25-26) should have four spaces on indention

**Comment by ericschmidtatwork** [comment]
Fixed lint issues, and I can add a README that links to the tutorial for this sample once the tutorial goes live.

**Comment by ericschmidtatwork** [comment]
@gguuss That template doesn't look like it applies here. The `main.py` file is not a script you run locally like that. I would use a simple README.md like the rest of the functions samples do.

**Comment by gguuss** [comment]
> @gguuss That template doesn't look like it applies here. The `main.py` file is not a script you run locally like that. I would use a simple README.md like the rest of the functions samples do.

Ack. I can see about adding it for you after you merge.

**Pull request merged**